code_wars/check_same_case.py

```diff
@@ -1 +1,10 @@
 # https://www.codewars.com/kata/5dd462a573ee6d0014ce715b
+def same_case(a, b):
+    case_result = 2
+    if isinstance(a, str) == True:
```

Comment:
`isinstance(a, str)` checks to see if the variable stored in `a` is the 'type' `str`

You can see this in a repl by using the builtin `type(<variable>)` function.

`"a"` and `"?"` are both strings, so everything in your tests will always be true on this condition.  You might checkout the string method  [isalpha](https://docs.python.org/3/library/stdtypes.html#str.isalpha)